indico/modules/events/registration/templates/display/_regform_info.html

```diff
@@ -7,13 +7,17 @@
                 <i class="icon icon-time"></i>
                 <div class="text">
                     <div>
-                        {% if regform.has_started %}
+                        {% if regform.has_started and not regform.has_ended %}
                             <span class="label">
                                 {% trans %}Registration is open{% endtrans %}
                             </span>
                         {% else %}
                             <span class="label">
-                                {% trans %}Opens{% endtrans %}
+                                {% if regform.has_ended %}
+                                    {% trans %}Opened{% endtrans %}
```

Comment:
i don't think it's really useful information for users to know when the registration period started

Reply:
It could be sometimes, for transparency reasons, e.g. if there are limited places for a conference and all get filled up quickly (just a quick thought). So you suggest to hide the schedule info altogether?

Reply:
Yes, I would show:

- that the registration is open and what the deadline is [if there's a closing time]
- no schedule at all if there is no deadline and the registration is open
- when the registration opens(/closes) if the registration is not open
- when the registration opened/closed if the registration is closed